e2e/test/scenarios/admin-2/whitelabel.cy.spec.js

```diff
@@ -89,25 +89,47 @@ describeEE("formatting > whitelabel", () => {
   });
 
   describe("favicon", () => {
-    beforeEach(() => {
-      cy.visit("/admin/settings/whitelabel");
-
-      cy.log("Add favicon");
-      cy.findByLabelText("Favicon").type(
-        "https://cdn.ecosia.org/assets/images/ico/favicon.ico",
-      );
-      cy.get("ul").eq(1).click("right");
-      // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-      cy.findByText("Saved");
-      checkFavicon();
-    });
-    it("should show up in user's HTML", () => {
+    it("should work for people that set favicon URL before we change the input to file input", () => {
```

Comment:
Add a test to ensure that users who have been using favicon URL would still have favicon working